### ISSUE TITLE
Some controls are not read correctly in settings UI with screen readers.

### ISSUE THREAD

**Issue by jareds** [microsoft/vscode#106975]
Issue Type: <b>Bug</b>

@isidorn 
1. Insure you are running with NVDA version 20.2. Other versions will probably show the same behavior.
2. Install the java extention pac.
3. Open the settings UI.
4. Go to extentions -> java
5. Tab through until you hear "edit in settings.json"
NVDA does not announce what I am actually goint to edit in settings.json if I click the button.

VS Code version: Code - Insiders 1.50.0-insider (9e505675670d65138405321a60b0df4ddec28799, 2020-09-16T06:49:37.816Z)
OS version: Windows_NT x64 10.0.19041
Remote OS version: Linux x64 4.19.84-microsoft-standard

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i3-7100U CPU @ 2.40GHz (4 x 2400)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.87GB (1.89GB free)|
|Process Argv|--crash-reporter-id 13f64f2b-f4fe-4a58-989d-cf59898848a6|
|Screen Reader|yes|
|VM|0%|

|Item|Value|
|---|---|
|Remote|Dev Container: Go|
|OS|Linux x64 4.19.84-microsoft-standard|
|CPUs|Intel(R) Core(TM) i3-7100U CPU @ 2.40GHz (4 x 2400)|
|Memory (System)|6.10GB (2.09GB free)|
|VM|0%|
</details><details><summary>Extensions (12)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-lombok|Gab|1.0.1
javadoc-tools|mad|1.4.0
remote-containers|ms-|0.141.0
remote-wsl|ms-|0.44.5
java|red|0.67.0
sonarlint-vscode|Son|1.17.0
vscodeintellicode|Vis|1.2.10
vscode-java-debug|vsc|0.28.0
vscode-java-dependency|vsc|0.13.0
vscode-java-pack|vsc|0.10.0
vscode-java-test|vsc|0.24.2
vscode-maven|vsc|0.24.2

</details>

**Comment by isidorn** [comment]
If I understand the issue correctly: Some complex settings we do not allow to edit in the settings editor, for that we have to poen the settings.json.
So the fix would be that the aria label of the "edit in settings.json" button also controls the setting name. So for example
"edit terminal.integrated.automationShell.linux in settings.json"

**Comment by jareds** [comment]
That is correct and the way you describe it is how I would expect it to
behave.
